### PR TITLE
Goods Selector

### PR DESCRIPTION
**Pull request by stunnerboyone**
[DEMO LINK](https://stunnerboyone.github.io/react_goods-selector-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,70 +16,72 @@ export const goods = [
...
-              data-cy="RemoveButton"
+              data-cy="ClearButton"
               type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+              className="delete ml-3"
+              onClick={() => setSelectedGood('')}
```
Comment (volodymyr-soltys97) [inline src/App.jsx:32]: Create a helper function for this logic and use them everywhere

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

File: src/App.jsx [hunk 2]
```diff
@@ -14,70 +16,76 @@ export const goods = [
...
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+              className="delete ml-3"
+              onClick={() => selectGoodOnClick()}
```
Comment (anastasiiavorobiova) [inline src/App.jsx:40]:
```suggestion
              onClick={selectGoodOnClick}
```

**Review by anastasiiavorobiova — APPROVED** [review]
Good work!

**Pull request closed without merging**